packages/amplify-codegen/tests/commands/models.test.js

```diff
@@ -0,0 +1,113 @@
+const generateModels = require('../../src/commands/models');
+const mock = require('mock-fs');
+const graphqlCodegen = require('@graphql-codegen/core');
+const fs = require('fs');
+const path = require('path');
+
+jest.mock('@graphql-codegen/core');
+const MOCK_CONTEXT = {
+  print: {
+    info: jest.fn(),
+  },
+  amplify: {
+    getProjectMeta: jest.fn(),
+    getEnvInfo: jest.fn(),
+    getResourceStatus: jest.fn(),
+    executeProviderUtils: jest.fn(),
+    pathManager: {
+        getBackendDirPath: jest.fn()
+    },
+    getProjectConfig: jest.fn()
+  },
+};
+const OUTPUT_PATHS = {
```

Comment:
The constants here is not linked to the source code and will break if the default path changes. I suggest to use mock frontend and mock ouput path.

Reply:
Yes. There are the expected output_paths where models should be generated for each frontend. If a change is made in the source files to change this behavior, it can be caught here in which case we will update the test too with new output path.